Fundamentals/Setup/Upgrading/umbraco10-breaking-changes.md

```diff
@@ -0,0 +1,149 @@
+---
+versionFrom: 10.0.0
+meta.Title: "Breaking changes going from Umbraco 9 to 10"
+meta.Description: "In this article we list the breaking changes between Umbraco 9 and 10"
+---
+
+# Breaking changes
+
+Below it is possible to read about the breaking changes between Umbraco 9 and 10.
+The article is subject to change, the closer we get to the launch of Umbraco 10.
+
+## Update 'diff' from 3.5.0 to 5.0.0
+
+diff library used in the Backoffice client is outdated by two major versions. Updating this will be a breaking change since the exposed global object has been renamed from JsDiff to Diff.
+
+## V10/feature/content schedule performance
+
+Removes mutable ContentSchedule property from IContent/Content to read/write content schedules please make use of _IContentService.GetContentScheduleByContentId && IContentService.PersistContentSchedule_ or the optional _contentSchedule parameter_ on _IContentService.Save_
+
+## v10 misc - Removed some redundant event handling code
+
+- Removed public methods: _PublishedSnapshotServiceEventHandler.Dispose_, _PublishedSnapshotServiceEventHandler.Dispose(bool)_ and _.PublishedSnapshotServiceEventHandler.Initialize_
+- Removed public ctor.
```

Comment:
This doesn't look right. Is it incomplete? If it's complete, maybe format it is `ctor`?

Reply:
Yeah, there is not more information about that specific breaking change unfortunately...